docs/vsto/visual-studio-tools-for-office-runtime.md

```diff
@@ -43,6 +43,6 @@ The following is a list of the VSTO Runtime releases and the corresponding versi
 | 10.0.60715 | 07/20/2015 | Microsoft Office 2016 |
 | 10.0.60724 | 07/24/2015 | Microsoft Office 2016 |
 | 10.0.60825 | 11/19/2016 | Microsoft Office 2016 |
-| 10.0.60828 | 01/12/2018 | Microsoft Office 2019 / <br>Office 365 |
+| 10.0.60828 | 01/12/2018 | Microsoft Office 2019 / <br>Office 365 /| <br>Office 2021 |
```

Comment:
I don't think you want the vertical bar between Office 365 and Office 2021 as that would try and add another column

Reply:
@abhinav2605  can you make the correction please?

Reply:
I have updated the change for vertical bar(pipe).